source/broadband/webpa_notification.c

```diff
@@ -1202,6 +1347,28 @@ void processNotification(NotifyData *notifyData)
 	        		cJSON_AddStringToObject(notifyPayload, "cid", cid);
 				OnboardLog("%s/%d/%s\n",dest,cmc,cid);
                                 WAL_FREE(cid);
+				memset(param_notify_string, 0, sizeof(param_notify_string));
+				if(cmc == 768)
```

Comment:
we can mention here a comment,
//sending parameter details when CMC is 768 during unknown value change event